lib/config/fileconf.go

```diff
@@ -321,6 +326,7 @@ type Global struct {
 	DataDir     string           `yaml:"data_dir,omitempty"`
 	PIDFile     string           `yaml:"pid_file,omitempty"`
 	AuthToken   string           `yaml:"auth_token,omitempty"`
+	AWSToken    AWSToken         `yaml:"aws_token,omitempty"`
```

Comment:
I chose the name as an alternative to `auth_token`, but I agree there is a difference because in the past a token **was** a string, now it has a set of rules that a node must prove it satisfies.

Really, this is the section used on the node to identify that it should use Simplified Node Joining. An alternative would be:
```
teleport:
  join_params:
    token_name: "example_simplified_join_token"
    type: ec2
```

I think this would be more extensible if we ever implement any other join methods, nothing at the top level is specific to AWS. @klizhentas any thoughts on the UX here?

Reply:
I went ahead and updated the file configuration to the above